code/game/objects/effects/glowshroom.dm

```diff
@@ -43,21 +50,30 @@
 		QDEL_NULL(myseed)
 	return ..()
 
-/obj/structure/glowshroom/New(loc, obj/item/seeds/newseed, mutate_stats)
+/**
+  *	Creates a new glowshroom structure.
+  *
+  * Arguments:
+  * * newseed - Seed of the shroom
+  * * mutate_stats - If the plant needs to mutate their stats
+  * * spread - If the plant is a result of spreading, reduce its stats
+  */
+
+/obj/structure/glowshroom/New(loc, obj/item/seeds/newseed, mutate_stats, spread)
```

Comment:
Any reason for this not to be `Initialize`?

Reply:
I'm not the author of the original function, so honestly I'm not sure which one should it be. Following https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#prefer-initialize-over-new-for-atoms I'm not sure if I should do this conversion 😕

Reply:
Please do use it, and don't forget to preserve the return of the parents call (`. = ..()` instead of just `..()`)

Reply:
Done, hope it's working. Tested locally and couldn't see issues.